src/InputGroupButton.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import Button from './Button';
+
+const propTypes = {
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  children: PropTypes.node,
+  groupClassName: PropTypes.any,
```

Comment:
Been thinking about these one off props. They seem fine for now. Do you think that something like `tags`, `props` and `classNames` would help define a common pattern for this kind of stuff? ex: `classNames={{group, button}}`, `props={{group: groupAttrs}}`.